src/components/FormType.tsx

```diff
@@ -79,7 +84,7 @@ const GitHubFormType = ({ modals }: GitHubFormTypeProps) => {
 
       // passing accessToken is necessary because DB access token can be stale which can cause 401: Bad Credentials
       const createWebhookResultObject = await createWebhook.mutateAsync({
-        accessToken: session.user.accessToken,
+        // accessToken: session.user.accessToken,
```

Comment:
why is this commented out, and what does that mean for the `// passing accessToken is necessary...`  comment above

Reply:
Reason is in test plan in pr template